src/main/java/com/cloudbees/jenkins/plugins/awscredentials/AWSCredentialsImpl.java

```diff
@@ -148,7 +147,7 @@ public AWSCredentials getCredentials() {
 
             AWSSecurityTokenService client;
             // Handle the case of delegation to instance profile
-            if (StringUtils.isBlank(accessKey) && StringUtils.isBlank(secretKey.getPlainText()) ) {
+            if (StringUtils.isBlank(accessKey) && StringUtils.isBlank(secretKey.getPlainText())) {
```

Comment:
Not sure what the maintainer’s preferences are, but FWIW I advise keeping PRs as short as possible, meaning in particular do not make unrelated formatting changes.

Reply:
Duly noted, I'll try not to do that in the future.

I'd un-format if I could, but I'm not sure how to do that.

(FWIW I'm using IntelliJ)

Reply:
> I'd un-format if I could, but I'm not sure how to do that.

Well you can manually reverse-apply the portions of the diff which do not change behavior.

There is probably some setting in your IDE to refrain from changing formatting of a line unless you actually edited that line. I do not use IDEA so I cannot say for sure.

Reply:
I'll mark this resolved for now. If someone (other than you) speaks up about the formatting, I'll manually change it back.